manager/apis/app/v1alpha1/cluster_selector.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterSelector is a label query over a set of resources in the specified cluster.
+type ClusterSelector struct {
+	// Cluster name
+	// +required
+	ClusterName string `json:"clusterName"`
```

Comment:
I see that clustername is required, but in your general comments didn't you say that if it's not provided you would take the information from the local cluster?

Reply:
Cluster selector itself may not be provided in the spec. If it is provided, it will have both fields defined.

Reply:
@shlomitk1  why not make this optional as well?

Reply:
OK.